package.json

```diff
@@ -5,7 +5,7 @@
   "license": "MIT",
   "author": "ReadMe <support@readme.io> (https://readme.com)",
   "engines": {
-    "node": ">=18"
+    "node": "^18.20.0 || >=20.10.0"
```

Comment:
Do our compiled dists contain these import assertion so? If not then this bump is really just for building it?

Reply:
they do yeah 😔 all these files contain the import attributes and make it into the compiled dist:

<img src="https://github.com/readmeio/rdme/assets/8854718/05b86641-d862-46d6-882e-386b64a53538" width=400>

Reply:
here's an example dist file:

![CleanShot 2024-04-24 at 17 39 53@2x](https://github.com/readmeio/rdme/assets/8854718/dbd082b0-5887-42e7-b8e5-1fb498223534)

Reply:
yep — it's just a single CLI dist so no `tsup` madness needed here thankfully